### DISCUSSION TITLE
New badge

### DISCUSSION THREAD

**Discussion by M9yk** [community/community#18757] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get the Galaxy Brain badge?

**Comment by jeezou** [comment, marked as answer]
Need to answer two discussions

https://github.com/Schweinepriester/github-profile-achievements

**Comment by ckc9759** [comment]
When you answer two discussions and they get accepted as "**Marked as answe**r" you get that badge !!
https://github.com/ckc1404 Still not got it lol !!
